crates/router/src/core/user.rs

```diff
@@ -1478,19 +1481,44 @@ pub async fn create_org_merchant_for_user(
     Ok(ApplicationResponse::StatusOk)
 }
 
+#[cfg(feature = "v1")]
 pub async fn create_merchant_account(
     state: SessionState,
     user_from_token: auth::UserFromToken,
     req: user_api::UserMerchantCreate,
-) -> UserResponse<()> {
+) -> UserResponse<api_models::admin::MerchantAccountResponse> {
```

Comment:
Import can be improved.
```suggestion
) -> UserResponse<admin_api::MerchantAccountResponse> {
```